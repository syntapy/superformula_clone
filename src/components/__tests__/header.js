import React from "react"
import renderer from "react-test-renderer"

import Header from "../header"

jest.mock("gatsby")

describe("Header", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<Header siteTitle="Default Starter" />)
			.toJSON()
		expect(tree).toMatchSnapshot()
	})
})
