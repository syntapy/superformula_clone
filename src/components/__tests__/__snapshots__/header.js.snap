// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="undefined undefined"
  >
    <SvgrURL />
    <div
      className="undefined undefined"
    >
      <a
        className="undefined undefined undefined"
        href="/"
      >
        Superformula.
      </a>
    </div>
    <nav
      className="undefined undefined"
    >
      <div
        className="undefined undefined undefined"
      >
        <a
          className="undefined undefined undefined undefined"
          href="/services"
        >
          Services
        </a>
      </div>
      <div
        className="undefined undefined undefined"
      >
        <a
          className="undefined undefined undefined undefined"
          href="/work"
        >
          Work
        </a>
      </div>
      <div
        className="undefined undefined undefined"
      >
        <a
          className="undefined undefined undefined undefined"
          href="/articles"
        >
          Articles
        </a>
      </div>
      <div
        className="undefined undefined undefined"
      >
        <a
          className="undefined undefined undefined undefined"
          href="/contact"
        >
          Contact
        </a>
      </div>
      <div
        className="undefined undefined undefined"
      >
        <a
          className="undefined undefined undefined undefined"
          href="https://careers.superformula.com"
        >
          Careers
        </a>
      </div>
    </nav>
  </div>
</header>
`;
